sadx-fixed-edition/mod.cpp

```diff
@@ -28,23 +103,33 @@ static PatchInfo patches[] = {
 	{ reinterpret_cast<void*>(0x005C0D5D), &CasinoSpawnY, sizeof(float) }
 };
 
-static PointerInfo jumps[] = {
-	// ItemBox
-	{ ItemBox_Display_Destroyed,           ItemBox_Display_Destroyed_Rotate },
-	{ ItemBox_Display_Unknown,             ItemBox_Display_Unknown_Rotate },
-	{ ItemBox_Display,                     ItemBox_Display_Rotate },
-	{ reinterpret_cast<void*>(0x0042CCC7), PlaySegaSonicTeamVoice_asm },
-	{ reinterpret_cast<void*>(0x0042CD2F), PlaySegaSonicTeamVoice_asm },
-};
-
 extern "C"
 {
 	EXPORT PatchList   Patches[]   = { { _arrayptrandlength(patches) } };
-	EXPORT PointerList Jumps[]     = { { _arrayptrandlength(jumps) } };
 	EXPORT ModInfo     SADXModInfo = { ModLoaderVer };
-
 	EXPORT void __cdecl Init(const char* path, const HelperFunctions& helperFunctions)
 	{
+		// Config stuff
```

Comment:
Below this point, several DLLs here are being checked for presence with `GetModuleHandle` multiple times. They should be checked once and their results stored in a local variable.